client/lib/plans/constants.js

```diff
@@ -208,8 +207,6 @@ export const PLANS_LIST = {
 			isEnabled( 'publicize-scheduling' ) && FEATURE_REPUBLICIZE_SCHEDULING,
 			FEATURE_BUSINESS_ONBOARDING,
 			FEATURE_ADVANCED_SEO,
-			isATEnabledForCurrentSite() && FEATURE_UPLOAD_PLUGINS,
-			isATEnabledForCurrentSite() && FEATURE_UPLOAD_THEMES,
```

Comment:
Currently this only displays for folks already with a Business plan.

We could maybe just use the abtest here to gate this, since we're only using it for the new user check.